radix-engine/src/engine/node_properties.rs

```diff
@@ -17,8 +17,13 @@ impl VisibilityProperties {
         actor: &ResolvedActor,
         node_id: RENodeId,
     ) -> bool {
-        if !mode.eq(&ExecutionMode::Application) {
-            return false;
+        match mode {
+            ExecutionMode::LoggerModule => match node_id {
+                RENodeId::Logger => return true,
+                _ => return false,
+            },
+            ExecutionMode::Application => {}
```

Comment:
This fall-through is not read-friendly. Possible to combine with the following match?

Reply:
hrm, it's not a fall-through as this match exits the match block whereas the following match returns immediately. Unless you mean something else?

Reply:
ah i got you. Yup sure will update this.